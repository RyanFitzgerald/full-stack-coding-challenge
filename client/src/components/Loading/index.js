import React from 'react';

import LoadingGif from '../../static/loading.gif';
import Wrapper from './Wrapper';

function Loading() {
  return (
    <Wrapper>
      <img src={LoadingGif} alt="Loading"/>
    </Wrapper>
  );
}

export default Loading;