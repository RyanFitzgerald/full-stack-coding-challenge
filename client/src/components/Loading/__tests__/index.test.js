import React from 'react';

import Loading from '../index';

describe('<Loading />', () => {
  it('should match the snapshot', () => {
    const renderedComponent = renderer.create(
      <Loading/>
    ).toJSON();
    
    expect(renderedComponent).toMatchSnapshot();
  });

  it('should contain loading image', () => {
    const renderedComponent = mount(
      <Loading/>
    );
    
    expect(renderedComponent.find('img')).toHaveLength(1);
  });
});