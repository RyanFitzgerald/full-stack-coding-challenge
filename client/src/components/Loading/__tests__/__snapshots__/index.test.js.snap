// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should match the snapshot 1`] = `
.c0 {
  margin: 30px 0;
  text-align: center;
}

<div
  className="c0"
>
  <img
    alt="Loading"
    src="loading.gif"
  />
</div>
`;
